repos/system_upgrade/common/actors/systemfacts/actor.py

```diff
@@ -41,3 +57,8 @@ def process(self):
         self.produce(systemfacts.get_selinux_status())
         self.produce(systemfacts.get_firewalls_status())
         self.produce(systemfacts.get_firmware())
+        self.produce(systemfacts.get_default_grub())
```

Comment:
```suggestion
        self.produce(systemfacts.get_default_grub_conf())
```

Reply:
and question is, in case the file doesn't exist on s390x, should be the msg produced? (someone should check s390x)

Reply:
Nope, it should not. Good catch!